modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -5013,19 +5013,13 @@ public void testGetObjectEntryByExternalReferenceCodeWithSlash()
 			).toString(),
 			_objectDefinition1.getRESTContextPath(), Http.Method.POST);
 
-		try (LogCapture logCapture = LoggerTestUtil.configureLog4JLogger(
-				"com.liferay.portal.vulcan.internal.jaxrs.exception.mapper." +
-					"WebApplicationExceptionMapper",
-				LoggerTestUtil.ERROR)) {
-
-			Assert.assertEquals(
-				405,
-				HTTPTestUtil.invokeToHttpCode(
-					null,
-					_objectDefinition1.getRESTContextPath() +
-						"/by-external-reference-code/a/b",
-					Http.Method.GET));
-		}
+		Assert.assertEquals(
```

Comment:
I think we should include more tests, mainly for the use case:

https://github.com/liferay/liferay-portal/blob/e97321d86948281eee624ac2ce525f835f09f09f/modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/BaseObjectEntryResourceImpl.java#L299

```
"/by-external-reference-code/{currentExternalReferenceCode}/{objectRelationshipName}/{relatedExternalReferenceCode}"
```

For example, given an object entry 1 with ERC `a/b`, a relationship called `c`, and a object entry 2 with ERC `d/e`, which is the expected behaviour?

With that, we should document the limitations, because I guess JAX-RS will map correctly in some cases and map incorrectly in some others.

Reply:
:+1: See https://github.com/liferay-headless/liferay-portal/pull/2037/commits/3cbef2fcb54e26f1c75556e5528793da3fe4a774.